helper/logging/logging.go

```diff
@@ -9,19 +10,21 @@ import (
 	"syscall"
 
 	"github.com/hashicorp/logutils"
+	testing "github.com/mitchellh/go-testing-interface"
 )
 
 // These are the environmental variables that determine if we log, and if
 // we log whether or not the log should go to a file.
 const (
-	EnvLog     = "TF_LOG"      // Set to True
-	EnvLogFile = "TF_LOG_PATH" // Set to a file
+	EnvLog         = "TF_LOG"      // Set to True
+	EnvLogFile     = "TF_LOG_PATH" // Set to a file
+	EnvLogPathMask = "TF_LOG_PATH_MASK"
```

Comment:
Should we include a comment here about its purpose for future travelers?

```suggestion
	// EnvLogPathMask splits test log files by name.
	EnvLogPathMask = "TF_LOG_PATH_MASK"
```